local_build/local_build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Locally build and compress the core Blockly files into a single JavaScript
+# file.
+#
+# Copyright 2018 Google Inc.
+# https://developers.google.com/blockly/
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Usage: local_build.sh.
+#
+# This script generates only local_blockly_compressed.js.  You may modify it as
+# needed to build other files.
+#
+# The compressed file is a concatenation of all of Blockly's core files, run
+# through a local copy of Google's Closure Compiler with simple optimizations
+# turned on.
+
+# Future work:
+# - Trim down Google's Apache licenses, to match the output of build.py.
+# - Generate other compressed files generated by build.py normally.
+
+# Find the Closure Compiler.
+if [ -f "$(npm root)/google-closure-compiler/compiler.jar" ]; then
+  # Travis test.
+  COMPILER="$(npm root)/google-closure-compiler/compiler.jar"
+elif [ -f *compiler*.jar ]; then
+  # Manual test.
```

Comment:
Echo the path/filename of the compiler selected, here or below.
Add TODO about checking only a single file was found.

Add the same to tests/compile/compile.sh
(And while there, correct indentation on line 9 of that file.)

Reply:
Done.